src/components/common/Alert/alert.styles.ts

```diff
@@ -1,28 +1,25 @@
+import { css } from "@emotion/react";
 import styled from "@emotion/styled";
-import { TABLET } from "../../../theme/common/breakpoints";
-import { Alert } from "./alert";
+import { Alert } from "@mui/material";
+import { SIZE } from "../../../styles/common/constants/size";
+import {
+  textBody4002Lines,
+  textBodyLarge500,
+} from "../../../styles/common/mixins/fonts";
 
-/**
- * Flat alert - typically used when in full stretch or full "bleed" across a container.
- * e.g. the entire width of mobile viewports.
- */
-export const FlatAlert = styled(Alert)`
-  border-left: none;
-  border-radius: 0;
-  border-right: none;
-  box-shadow: none;
-`;
-
-/* eslint-disable valid-jsdoc -- disable require param */
-/**
- * Fluid alert - typically used to transition between flat paper (mobile) and rounded paper (tablet or desktop).
- */
-/* eslint-enable valid-jsdoc -- disable require param */
-export const FluidAlert = styled(Alert)`
-  ${({ theme }) => theme.breakpoints.down(TABLET)} {
-    border-left: none;
-    border-radius: 0;
-    border-right: none;
-    box-shadow: none;
-  }
+export const StyledAlert = styled(Alert)`
```

Comment:
The `Alert` component has been simplified and the only styles now relate to size: see [mock](https://www.figma.com/design/JOUDXkKfgivRNFdBsWDPEH/AnVIL?node-id=587-5515&t=zhf66W28cJJO6DxI-4) and [mock](https://www.figma.com/design/JOUDXkKfgivRNFdBsWDPEH/AnVIL?node-id=587-5534&t=zhf66W28cJJO6DxI-4).